pkg/visor/visor.go

```diff
@@ -210,6 +213,12 @@ func NewNode(config *Config, masterLogger *logging.MasterLogger) (*Node, error)
 
 // Start spawns auto-started Apps, starts router and RPC interfaces .
 func (node *Node) Start() error {
+	skywireNetworker := appnet.NewSkywireNetworker(logging.MustGetLogger("skynet"), node.router)
+	appnet.ClearNetworkers()
```

Comment:
We shouldn't need to call `ClearNetworkers` here. We should either have an `init` function in `appnet` that initiates `networkers`, or we int `networkers` in `AddNetworker`.